src/middleware/arns.ts

```diff
@@ -108,3 +117,81 @@ export const createArnsMiddleware = ({
     res.header('Cache-Control', `public, max-age=${ttl}`);
     dataHandler(req, res, next);
   });
+
+export class BlockedNamesCache {
```

Comment:
@djwhitt Do you think this is the best place to have this class?

I have some questions about this class:
 1. If the cache is updated from the db when it starts and names are added and removed when we block and unblock names I don't think we need TTL and the setInterval to update the cache from the db.
 2. If 1 is correct we don't even need this class as the cache can be just a variable.

Reply:
[`95f516a` (#245)](https://github.com/ar-io/ar-io-node/pull/245/commits/95f516a24ea4498d67addb2668035d6f02c17d96)